docs/en/userguide/install.md

```diff
@@ -175,4 +175,16 @@ helm install fluid --set runtime.mountRoot=/var/lib/docker/runtime-mnt fluid
 
 ```
 helm install fluid --set csi.featureGates='FuseRecovery=true' fluid
-```
\ No newline at end of file
+```
+
+3. If your Kubernetes cluster has custom configured kubelet root-dir, please configure KUBELET_ROOTDIR when installing Fluid: 

```

Comment:
If your Kubernetes cluster has a custom configured kubelet root directory, please configure the KUBELET_ROOTDIR when installing Fluid with the following command:

Reply:
ok.